src/sentry/static/sentry/app/views/issueList/header.tsx

```diff
@@ -116,39 +133,30 @@ function IssueListHeader({
               return (
                 <li key={tabQuery} className={query === tabQuery ? 'active' : ''}>
                   <Link to={to} onClick={() => trackTabClick(tabQuery)}>
-                    <GuideAnchor
-                      target={inboxGuideStepOne ? 'inbox_guide_tab' : 'none'}
-                      disabled={!inboxGuideStepOne}
-                      to={to}
-                    >
-                      <GuideAnchor
-                        target={inboxGuideStepTwo ? 'for_review_guide_tab' : 'none'}
-                        disabled={!inboxGuideStepTwo}
+                    <WrapGuideTabs query={query} tabQuery={tabQuery} to={to}>
```

Comment:
`carryAssistantForward` doesn't actually work because `Link` from `react-router` we use in `<Button>`  converts the `to` query to a url and sets the `href` of the anchor tab, overriding the `#assistant` we were passing in `guideAnchor.tsx`. I remember it working but maybe there was a change that I missed, maybe it wasn't ever working 😞 

Putting `hash: '#assistant'`to this `to` query here is a better way to carry the hash forward between guides anyways. Still the guide doesn't trigger after you land on the `For Review` tab, might be because of the increased number of `requiredTargets` see the screen capture I added (changed the `to`, added `hash`)

https://user-images.githubusercontent.com/15015880/110443991-617a5900-8071-11eb-867f-45970e09148f.mp4